src/pages/guides/index.md

```diff
@@ -261,7 +262,7 @@ I/O Events generates two public-private key pairs and sign your event payload us
 I/O Events sends the 2 digital signatures as webhook request headers and they are available via the header fields
 `x-adobe-digital-signature-1` and `x-adobe-digital-signature-2` respectively. 
 
-I/O Events also sends 2 public keys corresponding to the private keys used to generate the digital signatures. These public keys which are stored in Cloud Front and are publicly accessible using the Cloud Front urls which look like `https://xxxxx.cloudfront.net/pub-key-<random-uuid>.pem`. I/O Events sends the public keys urls for both the keys and they are available via the the request header fields `x-adobe-public-key1-url` and  `x-adobe-public-key2-url` respectively.
+I/O Events also sends 2 public keys corresponding to the private keys used to generate the digital signatures. These public keys are stored in Cloud Front and are publicly accessible using our adobe domain adobeioevents.com. I/O Events sends only the relative paths of the public keys i.e. `/prod/keys/pub-key-<random-uuid>.pem` and they are available via the request header fields `x-adobe-public-key1-path` and  `x-adobe-public-key2-path` respectively.
```

Comment:
1. Can we render the domain as a link please, so that it stands out?
2. Let's change `adobe` -> `Adobe`

Reply:
> These public keys are stored in Cloud Front and are publicly accessible using our adobe domain adobeioevents.com.

I think it is irrelevant and an implementation detail that Cloud Front is used for storage. The important piece is that the domain to retrieve the keys from is `adobeioevents.com` and that the request header provides the relative path.